app/views/changeset/_risks.html.erb

```diff
@@ -1,9 +1,9 @@
-<% if pull_requests.nil? || pull_requests.empty? %>
+<% if changeset.risky_pull_requests.nil? || changeset.risky_pull_requests.empty? %>
```

Comment:
Would be nice if `changeset.risky_pull_requests` would always return an empty array, or use `blank?`.

Reply:
Since `changeset.risky_pull_requests` is also used in the else, we could DRY this up to use 
```
if pull_requests = changeset.risky_pull_requests.presence
...
```

Reply:
👍

Reply:
Putting an assignment in an `if` statement obfuscates the meaning here I think - especially since I want the array, not the boolean return value (which would look something like this: `if (risky_prs = changeset.risky_pull_requests).blank?` ). I'm happy to use `blank?` but I don't see the gain in obfuscating the meaning of the code, just to avoid one more attribute method call on `changeset`.

Reply:
`if pull_requests = changeset.risky_pull_requests.presence` would return array nor nil